.gitlab-ci.yml

```diff
@@ -325,6 +336,20 @@ pkg:nix:deploy:
     - master
     - /^v.*\..*$/
 
+pkg:nix:deploy:channel:
+  <<: *deploy-template
+  environment:
+    name: cachix
+    url: https://coq.cachix.org
+  only:
+    variables:
+      - $GITHUB_DEPLOY_KEY
```

Comment:
I suggest renaming to `CACHIX_DEPLOYMENT_KEY`. I've just added the value of the SSH key to our main repo settings (as a protected, cachix env only, variable).

Reply:
Looks good.